src/Dropdown/Dropdown.js

```diff
@@ -37,6 +37,12 @@ const Trigger = createComponent({
   `,
 });
 
+let isBlurable = false;
+
+const setBlurable = value => {
+  isBlurable = value;
+}
```

Comment:
Can you `useRef` for this? Thinking something like
```js
const blurable = useRef(false);

const setBlurable = value => {
  blurable.current = value;
}
```

Reply:
Yes ^ this unfortunately won't work with multiple dropdowns because it is global.